### DISCUSSION TITLE
board

### DISCUSSION THREAD

**Discussion by giscus** [Phil-Fan/Notebook#2] [General]
# board

PhilFan的笔记本

https://www.philfan.cn/

**Comment by Phil-Fan** [comment]
欢迎大家交流~ :laugh:

**Comment by ZhengliangDuanfang** [comment]
好像是新添加的，来坐坐

**Reply by Phil-Fan** [reply]
:smiley:

**Comment by donotknowsjtu** [comment]
打卡

**Comment by CDcooldown** [comment]
到此一游

**Reply by Phil-Fan** [reply]
:smiley:

**Reply by Phil-Fan** [reply]
让我猜猜你是谁

**Comment by Alexanderk3** [comment]
man,可以再上架自控不

**Reply by Phil-Fan** [reply]
迁移至 https://www.philfan.cn/Robotics/Control/Control-01-%E8%87%AA%E5%8A%A8%E6%8E%A7%E5%88%B6%E5%8E%9F%E7%90%86/

**Comment by lingzhiyxp** [comment]
自律给你自由🫡

**Reply by Phil-Fan** [reply]
:smiley:

**Comment by Slowist-Lee** [comment]
前辈嚎！刚刚才发现这个博客qwq

**Reply by Phil-Fan** [reply]
欢迎新朋友~

**Comment by Ponyboy2005** [comment]
有趣的灵魂

**Reply by Phil-Fan** [reply]
欢迎您来

**Comment by madeline-past** [comment]
笔记很棒~想问下Class-Finance-投资这部分笔记是某门课的大纲吗 :-)

**Reply by Phil-Fan** [reply]
感谢支持~~，是校内的《证券投资学》课程，这个笔记约等于没记录（之后会填坑的qaq

**Comment by IIIILing** [comment]
在催更催更
https://zju-eesuad.github.io/EEStUdy-Place/All%20In%20One/Website/
在这里发现的

**Reply by Phil-Fan** [reply]
欢迎~

**Comment by xuchongchong** [comment]
来 https://nbnn.net 分享见解，这里是一个言论自由的、有深度的交流论坛，以确保你的观点会得有识之士的响应。本论坛收录一切真诚、犀利、深刻的观点。

**Comment by pppe0102** [comment]
哇哦，偶然路过，这种记录好有意思，大赞！

**Reply by Phil-Fan** [reply]
感谢 😄

**Comment by Vamperd** [comment]
感觉这样的网站好酷，很多课程也都是看了前辈的笔记，想请问这种个人网站该怎么搭建呀

**Reply by Phil-Fan** [reply]
感谢 🚀 

可以搜索 [Mkdocs建站] 等关键词，我并没有怎么记录建站的流程
可以fork别人的网站，然后对照着一点点搭一个

**Comment by WHISTLE3LLL** [comment]
好棒的学长！在小红书为你的网站发了篇笔记阅读量一直在增加！

**Reply by Phil-Fan** [reply]
欢迎～

**Comment by whocarsCynthia** [comment]
好酷的人生

**Reply by Phil-Fan** [reply]
欢迎

**Comment by dream-oyh** [comment]
前来打卡！！
推一下我自己的博客：dream-oyh.pages.dev（还有很多不完善的地方，无限进步啦！！！）

**Reply by Phil-Fan** [reply]
欢迎您来～

**Comment by DrRedTheRed** [comment]
好帅的博客

**Reply by Phil-Fan** [reply]
感谢～

欢迎大佬

**Comment by HHookUrbanB** [comment]
谁懂 在GitHub上找项目，突然发现了熟悉的头像和用户名 我打开了新大陆
